translations/en_GB/atrium-translations-en_GB-jvm/src/main/kotlin/ch/tutteli/atrium/translations/DescriptionPathAssertion.kt

```diff
@@ -24,6 +24,8 @@ enum class DescriptionPathAssertion(override val value: String) : StringBasedTra
     EXECUTABLE("executable"),
     A_FILE("a file"),
     A_DIRECTORY("a directory"),
+    AN_EMPTY_DIRECTORY("an empty directory"),
+    DIRECTORY_HAS("directory has"),
```

Comment:
I feel that ‘directory contains’ would be the better wording.

Reply:
@mai2412 the build currently fails because there is no such entry defined for de_CH. I also suggest that you change this to `directory contains`
```suggestion
    DIRECTORY_CONTAINS("directory contains"),
```
And then add the two entries to translations/de_CH/atrium-translations-de_CH-jvm/src/main/kotlin/ch/tutteli/atrium/translations/DescriptionPathAssertion.kt where you can use the following:
```
 AN_EMPTY_DIRECTORY("ein leeres Verzeichnis"),
DIRECTORY_CONTAINS("Verzeichnis enthält"),
```